### ISSUE TITLE
Import commits from several email addresses

### ISSUE THREAD

**Issue by as0n** [IonicaBizau/git-stats-importer#11]
The importer in it's current state only imports commits being written by an author with the exact email from `git config user.email`. This is fine in most cases but it doesn't account for accidental misspelling or authors changing email address over time.

Hence in the following exemple only the first of the two commits will be imported to git-stats : 
```
$ git config --global user.email
new@address.com
```
```
commit 015cff6fb440725c67638ad520c3b2a55cf83fc5
Author: Me <new@address.com>
Date:   Thu Feb 12 11:17:44 2015 +0200

    Stuff

commit 447eeae07d3307410e2c4d3484a2481240a4eb7f
Author: Me <old@address.com>
Date:   Thu Feb 12 11:17:18 2015 +0200

    Old stuff

```

Fortunately `git log` supports the use of `.mailmap` files (described [here](https://www.kernel.org/pub/software/scm/git/docs/git-shortlog.html#_mapping_authors)) which fixes this exact issue directly in git.

I ran into this issue myself and wrote a fairly simple implementation of `.mailmap` support for git-stats-importer. I will create a pull requests immediatly.

**Comment by IonicaBizau** [comment]
:+1:

**Comment by IonicaBizau** [comment]
This was fixed via #12.
